bootstrap

```diff
@@ -898,11 +899,14 @@ for d in $modules ; do
     xapian-core)
       cd "$d" && "$configure" --enable-maintainer-mode --disable-option-checking --cache-file="$here/config.cache" ${1+"$@"}
       ;;
+    xapian-letor)
+      cd "$d" && "$configure" --enable-maintainer-mode --disable-option-checking --cache-file="$here/config.cache" ${1+"$@"}
```

Comment:
You need to copy the old `*)` case here - currently `XAPIAN_CONFIG` isn't passed (which is the cause of the build failure you asked about on IRC).

Reply:
fixed in a fixup commit.